docs/volto_customization/custom_block.md

```diff
@@ -9,9 +9,9 @@ myst:
 
 # Volto Weather Block (custom block)
 
-Let's create a volto block that will display weather information for Eibar. For this we can use <a target="_blank" href="https://open-meteo.com/">Open-Meteo API</a>. Open-Meteo is an open-source weather API and offers free access for non-commercial use. No API key required.
+Let's create a volto block that will display weather information for Brasilia. For this we can use `<a target="_blank" href="https://open-meteo.com/">Open-Meteo API</a>`. Open-Meteo is an open-source weather API and offers free access for non-commercial use. No API key required.
```

Comment:
I totally misinterpreted this as being an inline literal to insert into a block. Sorry about the mistake.

Also I don't know why you use HTML markup when Markdown is easier, unless you want to force participants to open links in a new window with `target="_blank"`. This is the only training that does that. For this link and all others, if you want to use Markdown syntax, the pattern is `[label](URL)`.

```suggestion
Let's create a volto block that will display weather information for Brasilia. For this we can use [Open-Meteo API](https://open-meteo.com/). Open-Meteo is an open-source weather API and offers free access for non-commercial use. No API key required.
```

Reply:
Sorry, My bad. There're also other dangling links since last time. I'll remove them. I agree markdown is better.

Reply:
fixed in [a650478](https://github.com/plone/training/pull/860/commits/a650478b5d4c53a64428848e2936994cc45866c4)